terminal/tcell/event.go

```diff
@@ -0,0 +1,192 @@
+	tcellSpaceKey:           keyboard.KeySpace,
+	tcell.KeyF1:             keyboard.KeyF1,
+	tcell.KeyF2:             keyboard.KeyF2,
+	tcell.KeyF3:             keyboard.KeyF3,
+	tcell.KeyF4:             keyboard.KeyF4,
+	tcell.KeyF5:             keyboard.KeyF5,
+	tcell.KeyF6:             keyboard.KeyF6,
+	tcell.KeyF7:             keyboard.KeyF7,
+	tcell.KeyF8:             keyboard.KeyF8,
+	tcell.KeyF9:             keyboard.KeyF9,
+	tcell.KeyF10:            keyboard.KeyF10,
+	tcell.KeyF11:            keyboard.KeyF11,
+	tcell.KeyF12:            keyboard.KeyF12,
+	tcell.KeyInsert:         keyboard.KeyInsert,
+	tcell.KeyDelete:         keyboard.KeyDelete,
+	tcell.KeyHome:           keyboard.KeyHome,
+	tcell.KeyEnd:            keyboard.KeyEnd,
+	tcell.KeyPgUp:           keyboard.KeyPgUp,
+	tcell.KeyPgDn:           keyboard.KeyPgDn,
+	tcell.KeyUp:             keyboard.KeyArrowUp,
+	tcell.KeyDown:           keyboard.KeyArrowDown,
+	tcell.KeyLeft:           keyboard.KeyArrowLeft,
+	tcell.KeyRight:          keyboard.KeyArrowRight,
+	tcell.KeyEnter:          keyboard.KeyEnter,
+	tcell.KeyCtrlA:          keyboard.KeyCtrlA,
+	tcell.KeyCtrlB:          keyboard.KeyCtrlB,
+	tcell.KeyCtrlC:          keyboard.KeyCtrlC,
+	tcell.KeyCtrlD:          keyboard.KeyCtrlD,
+	tcell.KeyCtrlE:          keyboard.KeyCtrlE,
+	tcell.KeyCtrlF:          keyboard.KeyCtrlF,
+	tcell.KeyCtrlG:          keyboard.KeyCtrlG,
+	tcell.KeyCtrlJ:          keyboard.KeyCtrlJ,
+	tcell.KeyCtrlK:          keyboard.KeyCtrlK,
+	tcell.KeyCtrlL:          keyboard.KeyCtrlL,
+	tcell.KeyCtrlN:          keyboard.KeyCtrlN,
+	tcell.KeyCtrlO:          keyboard.KeyCtrlO,
+	tcell.KeyCtrlP:          keyboard.KeyCtrlP,
+	tcell.KeyCtrlQ:          keyboard.KeyCtrlQ,
+	tcell.KeyCtrlR:          keyboard.KeyCtrlR,
+	tcell.KeyCtrlS:          keyboard.KeyCtrlS,
+	tcell.KeyCtrlT:          keyboard.KeyCtrlT,
+	tcell.KeyCtrlU:          keyboard.KeyCtrlU,
+	tcell.KeyCtrlV:          keyboard.KeyCtrlV,
+	tcell.KeyCtrlW:          keyboard.KeyCtrlW,
+	tcell.KeyCtrlX:          keyboard.KeyCtrlX,
+	tcell.KeyCtrlY:          keyboard.KeyCtrlY,
+	tcell.KeyCtrlZ:          keyboard.KeyCtrlZ,
+	tcell.KeyBackspace:      keyboard.KeyBackspace,
+	tcell.KeyTab:            keyboard.KeyTab,
+	tcell.KeyEscape:         keyboard.KeyEsc,
+	tcell.KeyCtrlBackslash:  keyboard.KeyCtrlBackslash,
+	tcell.KeyCtrlRightSq:    keyboard.KeyCtrlRsqBracket,
+	tcell.KeyCtrlUnderscore: keyboard.KeyCtrlUnderscore,
+	tcell.KeyBackspace2:     keyboard.KeyBackspace2,
+	tcell.KeyCtrlSpace:      keyboard.KeyCtrlSpace,
+}
+
+// convKey converts a tcell keyboard event to the termdash format.
+func convKey(event *tcell.EventKey) terminalapi.Event {
+	tcellKey := event.Key()
+
+	if tcellKey == tcell.KeyRune {
+		ch := event.Rune()
+		return &terminalapi.Keyboard{
+			Key: keyboard.Key(ch),
+		}
+	}
+
+	k, ok := tcellToTd[tcellKey]
+	if !ok {
+		return terminalapi.NewErrorf("unknown keyboard key '%v' in a keyboard event", tcellKey)
```

Comment:
(optional) Would it be better to print event.Name() instead? Not sure which one would be more readable.

https://github.com/gdamore/tcell/blob/de7e78efa4a71b3f36c7154989c529dbdf9ae623/key.go#L206